Monika After Story/game/script-topics.rpy

```diff
@@ -7010,3 +7010,39 @@ label monika_dating_startdate_confirm(first_sesh_raw):
 
     # default action is to loop here
     jump .loopstart
+
+init 5 python:
+     addEvent(Event(persistent.event_database, eventlabel="monika_first_sight_love", category=["life", "romance"], prompt="Love at first sight", random=True))
+
+label monika_first_sight_love:
+    m 1d "Have you thought about the concept of 'love at first sight'?"
+    m 3d "Like, seeing someone for the first time, and instantly know they will be the love of life."
+    m 2q "I think it's one of the more ridiculous concepts to grasp."
+    m 2o "I mean, you can't just know who a person truly is, just by looking at them once."
+    m 2f "It's not like you talked to that person, had lunch or hung out together. "
```

Comment:
Personal opinion, I feel like Monika would choose the word "spoke" instead of "talked".

Reply:
I disagree. I feel as though I would hear talked instead of spoke in a normal conversation. Such as "hey, when we talked about your present before..." or "You should have talked to them then..."